src/main/java/com/google/sps/servlets/EmailServlet.java

```diff
@@ -0,0 +1,168 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that sends emails to the author and followers of an answered question or a commented
+ * answer.
+ */
```

Comment:
Will this Servlet also be used for sending emails during the mentor verification step? If so, maybe change this comment to be more generalized.

```suggestion
/**
 * Servlet that sends emails to users upon notification events, such as a followed question being answered. 
 */
```

Reply:
It will eventually be used for that, but because that is still not set up I was planning on adding that later. Do you recommend writing it in advance?

Reply:
No need to implement that now, but you can leave the top level documentation more generalized here. The most important thing to know about this servlet is that is sends emails to users upon notification events.